external/loaders/loaders/mappers/_hybrid.py

```diff
@@ -213,7 +70,7 @@ def _open_nudged_hybrid_portion(fine_shifted: xarray.Dataset, nudge_url: str):
     merged["latitude"] = merged.latitude.isel(time=0)
     merged["longitude"] = merged.longitude.isel(time=0)
 
-    return merged.astype(numpy.float32)
+    return merged.astype(np.float32)
 
 
 @mapper_functions.register
```

Comment:
To avoid too much scope creep I did not also add `approach` to the hybrid opener, so effectively the `apparent_sources_only` option is the only one for the hybrid, which seems fine for now.